.gitignore

```diff
@@ -8,4 +8,7 @@ test/tmp/
 *.log
 
 #Jira Connection Information
-.jirarc
\ No newline at end of file
+.jirarcTo 
+
+#DS_ Store
+.DS_Store
```

Comment:
@mtscout6 suggests adding `.DS_Store` to your global `.gitignore` rather than here.